src/calculateRentalCost.js

```diff
@@ -5,6 +5,27 @@
  */
 function calculateRentalCost(days) {
   // write code here
+  const dailyRent = 40;
+  const shortTermDiscount = 20;
+  const longTermDiscount = 50;
+
+  const totalRentPrice = days * dailyRent;
+
+  if (days === 0) {
+    return 0;
+  }
+
+  if (days < 3) {
+    return totalRentPrice;
+  }
+
+  if (days >= 3 && days < 7) {
+    return totalRentPrice - shortTermDiscount;
+  }
+
+  if (days >= 7) {
```

Comment:
You can remove the condition `days >= 7` here because if the code execution has reached this point, it means none of the previous conditions were met, and it is guaranteed that `days` is at least `7`.